pkgs/development/perl-modules/Bio-Ext-Align/default.nix

```
@@ -0,0 +1,21 @@
+{ config, lib, pkgs, buildPerlPackage, fetchzip }:
+
+buildPerlPackage {
+  pname = "BioExtAlign";
+  version = "1.5.1";
+  src = fetchzip {
+    # No longer maintained
+    url = "https://github.com/bioperl/bioperl-ext/archive/refs/tags/bioperl-ext-release-1-5-1.zip";
+    sha256 = "1h2l4m6r6ij11xhl41yc7f6fdqlclypvxdk61nyfj5fgmpm5jjzv";
+  };
+  patches = [ ./fprintf.patch ];
+  # PREFIX is important
+  configurePhase = ''
+    cd Bio/Ext/Align
+    perl -pi -e"s|(cd libs.+)CFLAGS=\\\'|\$1CFLAGS=\\\'-fPIC |" Makefile.PL
+    perl Makefile.PL PREFIX=$out INSTALLDIRS=site
+    '';
```

```suggestion
  configurePhase = ''
    cd Bio/Ext/Align
    perl -pi -e"s|(cd libs.+)CFLAGS=\\\'|\$1CFLAGS=\\\'-fPIC |" Makefile.PL
    perl Makefile.PL PREFIX=$out INSTALLDIRS=site
  '';
```
Why are we not using the default perl phasesm variable to extend them or pre/postConfigure? Same for buildPhase and installPhase.